packages/dashmate/src/status/scopes/services.js

```diff
@@ -0,0 +1,56 @@
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+module.exports = (dockerCompose) => async (config) => {
```

Comment:
We create factories a little bit differently. Please look at JS Drive:


servicesFactory.js
```js
/**
 * @returns {services}
 */
function servicesFactory(dep) {
  /**
   * @typedef services
   */ 
  function services() {
    ...
  }

  return services;
}

module.exports = servicesFactory;
```